src/scene/materials/standard-material.js

```diff
@@ -130,16 +130,18 @@ const _tempColor = new Color();
  * @property {string} specularityFactorVertexColorChannel Vertex color channels to use for specularity factor. Can be
  * "r", "g", "b", "a", "rgb" or any swizzled combination.
  * @property {boolean} enableGGXSpecular Enables GGX specular. Also enables
- * {@link StandardMaterial#anisotropy}  parameter to set material anisotropy.
- * @property {number} anisotropy Defines amount of anisotropy. Requires
+ * {@link StandardMaterial#anisotropyIntensity}  parameter to set material anisotropy.
+ * @property {number} anisotropyIntensity Defines amount of anisotropy. Requires
  * {@link StandardMaterial#enableGGXSpecular} is set to true.
- *
- * - When anisotropy == 0, specular is isotropic.
- * - When anisotropy < 0, anisotropy direction aligns with the tangent, and specular anisotropy
- * increases as the anisotropy value decreases to minimum of -1.
- * - When anisotropy > 0, anisotropy direction aligns with the bi-normal, and specular anisotropy
- * increases as anisotropy value increases to maximum of 1.
- *
+ * - When anisotropyIntensity == 0, specular is isotropic.
+ * - Specular anisotropy increases as anisotropyIntensity value increases to maximum of 1.
+ * @property {number} anisotropyRotation Defines the rotation of anisotropy. The value is in radian.
```

Comment:
Every other angle passed in the PlayCanvas Engine API is in degrees, so this looks strange to me. Any reason not to use degrees?

Reply:
> Every other angle passed in the PlayCanvas Engine API is in degrees,.
I didn't know about that, I'll change it to degrees.

Reply:
Fixed at https://github.com/playcanvas/engine/pull/7668/commits/052baff38e0e8d473322f391d534203da5ea75a0